superset-frontend/src/dashboard/actions/sliceEntities.js

```diff
@@ -38,96 +44,142 @@ export function fetchAllSlicesFailed(error) {
   return { type: FETCH_ALL_SLICES_FAILED, payload: { error } };
 }
 
-export function getDatasourceParameter(datasourceId, datasourceType) {
-  return `${datasourceId}__${datasourceType}`;
+export function fetchSlices(
+  userId,
```

Comment:
Could we send these as an `Option object`? So we don't have to worry about the order we send it and we avoid sending `undefined` when calling it? So we just send what options we need and we destructure here to get them?